Gruntfile.js

```diff
@@ -11,6 +11,15 @@ module.exports = function(grunt) {
           'frontend/src/**/lang/en.json'
         ],
         dest: 'routes/lang/en.json'
+      },
+      es: {
+        src: [
+          'routes/lang/en-application.json',
```

Comment:
I have built on top of the English files that way if things are ever missing from the Spanish file it will at least display it in english.